core/common/src/LocalTime.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2019-2022 JetBrains s.r.o.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime
+
+import kotlinx.datetime.serializers.LocalTimeIso8601Serializer
+import kotlinx.serialization.Serializable
+
+/**
+ * The time part of [LocalDateTime].
+ *
+ * This class does not describe specific *moments in time*, which are represented as [Instant] values.
+ * Instead, its instances can be thought of as clock readings, something that an observer in a particular time zone
+ * could witness.
+ * For example, `18:43` is not a *moment in time*, since someone in Berlin and someone in Tokyo would witness
+ * this on their clocks at different times.
+ *
+ * The arithmetic on [LocalTime] values is not provided, since without accounting for the time zone
+ * transitions or things like daylight saving or leap seconds it may give misleading results.
```

Comment:
Daylight saving and (in some time scales) leap seconds are special cases of time zone transitions, so
```suggestion
 * transitions it may give misleading results.
```